tests/api/test_serializer.py

```diff
@@ -147,7 +147,7 @@ def test_get_alias_infos_with_pagination_v3_no_duplicate_when_empty_contact(
     """
     Make sure an alias is returned once when it has 2 contacts that have no email log activity
     """
-    user = create_user(flask_client)
+    user = create_random_user()
```

Comment:
ah do you know why this test fails with `create_user`?

Reply:
ah, if I run the tests several times in my local installation, since the user has already been created the index fails.